akka-actor/src/main/scala/akka/actor/AbstractProps.scala

```diff
@@ -26,7 +25,7 @@ private[akka] trait AbstractProps {
     if (Modifier.isAbstract(clazz.getModifiers)) {
       throw new IllegalArgumentException(s"Actor class [${clazz.getName}] must not be abstract")
     } else if (!classOf[Actor].isAssignableFrom(clazz) &&
-               !classOf[IndirectActorProducer].isAssignableFrom(clazz)) {
+      !classOf[IndirectActorProducer].isAssignableFrom(clazz)) {
```

Comment:
Keeping if condition terms on new lines indented would make it more readable

Reply:
Yeah I am not sure what is causing this, maybe the alignment of `!` is messing with it and removing it would help

Reply:
So I just removed the  vertical alignment `!` operator however it didn't actually update this so it must be something else.

Reply:
I rather prefer the change, because I don't like branches with fruits danging on the right side 🙂 but YMMV.

Reply:
In any case I honestly don't see how I can fix this, I mean I can try experimenting with scalafmt options but it takes a large amount of time and I think what we have now is a good approximation.

Reply:
In that case I'm fine with it.